golem-cli/src/worker.rs

```diff
@@ -168,6 +168,29 @@ pub enum WorkerSubcommand {
         #[arg(short, long)]
         worker_name: WorkerName,
     },
+    /// Retrieves metadata about an existing workers in a template
+    #[command()]
+    List {
+        /// The Golem template the workers to be retrieved belongs to
+        #[command(flatten)]
+        template_id_or_name: TemplateIdOrName,
+
+        // Filter
+        #[arg(short, long)]
+        filter: Option<Vec<String>>,
+
+        // Cursor
```

Comment:
Please add some details here (i.e. "skip this amount of records in result" or something)